src/js/components/AppComponent.jsx

```diff
@@ -26,6 +26,19 @@ var AppComponent = React.createClass({
     );
   },
 
+  getStatus: function () {
+    var model = this.props.model;
+    var status = "Running";
+
+    if (model.deployments.length > 0) {
+      status = "Deploying";
+    } else if (model.instances === 0 && model.tasksRunning === 0) {
```

Comment:
How does this cope with https://github.com/mesosphere/marathon-ui/commit/0a35444cdd10348e4f34bee184e870e43cdcd2d3 ?